ckanext/data_qld/resource_freshness/helpers/helpers.py

```diff
@@ -0,0 +1,39 @@
+import datetime as dt
+import json
+
+import logging
+
+from ckan.lib.base import config
+
+log = logging.getLogger(__name__)
+
+update_frequencies = {
+    "monthly": 30,
+    "quarterly": 91,
+    "semiannually": 182,
+    "annually": 365
+}
+
+
+def get_update_frequencies():
+    try:
+        return json.loads(update_frequencies_from_config())
+    except ValueError:
+        log.warning('Unable to load update frequencies from config')
+        return update_frequencies
+
+
+def update_frequencies_from_config():
+    return config.get('ckanext.dataqld.update_frequencies')
```

Comment:
Update config value to match the name of the extension and namespace path eg. ckanext.data_qld.resource_freshness.update_frequencies.
Also there is already a metadata field name `update_frequency` which could get confused with this, should we rename the variables and method names to represent something more related to the functionality? Something like `update_frequency_days`?